### ISSUE TITLE
cross region secrets

### ISSUE THREAD

**Issue by dcarbonetti** [amazon-connect/amazon-connect-salesforce-scv#52]
not sure if this is an issue or not. I installed the Voicemail Express solution from the template on the 24/25th Jan and on finally getting around to testing it the packager seems to fail. modifying the Lambda to print the error I found the user was having auth issues on getting access to the secrets. 

Working with AWS they found on the cloudtrail it was trying to access the secrets in the US-West-2 region (Oregon) instead of AP-East-2 (Sydney)

To test this further we modified the Role to provide access to all resources in secrets which led to a resourceNotFound exception

TL;DR: seems the packager may be trying to access the secrets from the Oregon region instead of dynamically looking at the region the solution was set up in

**Comment by dcarbonetti** [comment]
Not a dev so sorry if i've got this wrong.

digging through the python common layers; file : sf_auth.py Line 10 ->11

`client = boto3.client(
    service_name='secretsmanager',
    region_name='us-west-2'
)`

I think that may be the issue @SteveEarl-AWS

**Comment by SteveEarl-AWS** [comment]
@dcarbonetti Thank you for reaching out.  That issue was resolved in commit a07837ed53f1140dc2b8785e85b7e376f90cb28c.

**Comment by dcarbonetti** [comment]
Thanks @SteveEarl-AWS appreciate your guidance
I've gone to try and do an update but it seems the instructions are out of date, or the deployments directory needs to be re-added to the repo; is there any other way to access the zip files

**Comment by SteveEarl-AWS** [comment]
@dcarbonetti Please be certain to remove the existing stacks and re-create them.  Do not simply update the stacks.
